src/JwtUtil.js

```diff
@@ -0,0 +1,13 @@
+import jwt from "jwt-decode";
+
+function isJwtExpired() {
+  const jwtToken = window.sessionStorage.getItem("access-token");
+  if (jwtToken === "" || jwtToken === undefined || jwtToken === null) {
+    return true;
+  } else {
+    var jwtExpired = jwt(jwtToken).exp * 1000 < new Date().getTime();
```

Comment:
what logic is this?

Reply:
I didnt understand

Reply:
we parse jwt token with jwt(tokenString), get expiry, multiply by 1000 to get milliseconds, and compare it with system time

Reply:
if system time is more than jwt time that equals jwt expired

Reply:
That I understood, what about the conditional logic
System time is the current time?

Reply:
yes

Reply:
and expiry is what??

Reply:
it says in how many seconds it expires??

Reply:
it is a unix time string, says at what date and time it expires

Reply:
Oh ok

Reply:
exp wont be like 2 hours or like that, it will be equal to jwt generation time + jwt validity time